source/common/config/utility.h

```diff
@@ -269,6 +269,11 @@ class Utility {
     return config;
   }
 
+  /**
+   * Truncates the message to a length less than default GRPC trailers size limit (by default 8KiB).
+   */
+  static std::string truncateDebugString(absl::string_view error_message);
```

Comment:
Given that this is now applied differently, perhaps a better name would be `truncateGrpcStatusMessage`?